kie-plugins-testing/src/test/resources/logback-test.xml

```diff
@@ -12,6 +12,7 @@
 
   <logger name="org.drools.compiler.xpath.tobeinstrumented" level="INFO"/>
   <logger name="org.kie" level="INFO"/>
+  <logger name="org.drools.ancompiler" level="debug"/>
```

Comment:
are you sure you want to leave this as `debug`?
```suggestion
  <logger name="org.drools.ancompiler" level="INFO"/>
```

Reply:
Yes it doesn't generate much output